src/reconcile/rec_write.c

```diff
@@ -3286,6 +3285,17 @@ __rec_split_write(WT_SESSION_IMPL *session,
 		}
 	}
 
+	bnd->entries = r->entries;
+	/* Output a verbose message if we create a page without many entries */
+	if (WT_VERBOSE_ISSET(session, WT_VERB_SPLIT) && r->entries < 6)
+		WT_ERR(__wt_verbose(session, WT_VERB_SPLIT,
+		    "Reconciliation creating a page with %" PRIu32
+		    " entries, memory footprint %" PRIu64
+		    ", page count %" PRIu32 ", %s, split state: %d\n",
+		    r->entries, r->page->memory_footprint, r->bnd_next,
+		    F_ISSET(r, WT_EVICTING) ? "evict" : "checkpoint",
+		    r->bnd_state));
+
```

Comment:
@agorrod, I don't think I've run with `WT_VERB_SPLIT` in years -- I used it long ago when debugging the original split code, and it's been pretty useless since then.

Instead of creating another level of naming by adding unique identifiers to verbose messages, why not create a new category of verbose messages? It doesn't cost us any performance to have more verbose message types, and the current implementation supports around 40 more verbose message types without any change.

There's also `WT_VERB_TEMPORARY`: you could leave the verbose message in the source using verb-temporary, but ifdef'd out, and it's trivial to turn it on whenever you want it?